Java

```diff
@@ -47,34 +58,27 @@ class student{
 		
 		
 		
-		
       	for(int t=0; t<w; t++) {
       	
           
         	System.out.print("Attendance for week # "+(t+1)+": ");
-        	time[t] = s.nextDouble(); 
-        
+        	double hours = s.nextDouble(); 
+        	students[i].addHours(t,hours);
+      	}
+	
         	
-      }
+  	
+        for(int t=0; t<n; t++){
 
-
-        
-        for(int t=0; t<w; t++){
-
-          total = total + time[t];
-
-        }          
-        double average = total / w;
-        double perc = average*10;
-       
+       students[i].addHours(double totalHours);
 	System.out.println(newline);
 		
-		System.out.println("Total hours present: "+total+" Hours");
-		System.out.println("Total hours conducted: "+(4*w)+" Hours");
-		System.out.printf("Overall attandence is : %.2f%%%n", perc);
+		System.out.println("Total hours present: "+ students[t].addHours()+ "Hours");
```

Comment:
Since we want to retrieve the number of attended hours for a student, what we want to do here `System.out.println("Total hours present: " + students[i].hours + " Hours");